src/main/java/uk/gov/hmcts/ccd/domain/types/ValidationContext.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.hmcts.ccd.domain.types;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseFieldDefinition;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseTypeDefinition;
+import uk.gov.hmcts.ccd.domain.model.std.CaseDataContent;
+
+import java.util.Map;
+
+public class ValidationContext {
+
+    private final CaseDataContent currentCaseDataContent;
+    private final String caseTypeId;
+    private final CaseTypeDefinition caseTypeDefinition;
+    private final Map<String, JsonNode> data;
+    private String fieldId;
+    private String path;
+    private JsonNode dataValue;
+    private CaseFieldDefinition caseFieldDefinition;
+
+    public ValidationContext() {
+        this(null, null, null, null);
+    }
```

Comment:
not used

Reply:
It is used by  Test for  previous Validators which does not use  ValidationContext

Reply:
DONE